### DISCUSSION TITLE
Alternative set of pieces for Chak

### DISCUSSION THREAD

**Discussion by Coatll** [gbtami/pychess-variants#1224] [Ideas]
Hi, 
I can see that Chak has only one set of pieces, and their style doesn't quite feel to me to fit the setting. So I made another one, trying to make the pieces similar to Mayan glyphs and also suggestive for the movement rules. 

![the glyph pieces](https://raw.githubusercontent.com/Coatll/Chak/main/chakPiecesNames1.png?token=GHSAT0AAAAAACIEASUHDQOZDVXR5SC7RLYSZIUXKNQ)

The points and lines at the sides of the icons indicate the rules of movement. The pictographs in the icon centers help identify the pieces.

Let me explain the identification:

- King is a variation of the real glyph meaning "King"
- pawn is actually a part of the Mayan glyph meaning "man". (According to other source, it means "monkey".)
- jaguar is a real jaguar glyph variation
- serpent and vulture are also real mayan glyphs. (They actually mean something different, but I chose them because they look like a serpent and a flying bird.)
- the two curves for shaman are meant as their face-paint or so
- quetzal is made up, the symbol represents the long tail of the bird
- offering is a real glyph for flint knife, which is a mesoamerican symbol of sacrifice.

This is how it looks on a board. I have made two slightly different versions, Can't decide which one is better.

![version 1](https://raw.githubusercontent.com/Coatll/Chak/main/chak1.png?token=GHSAT0AAAAAACIEASUGYZZ7DQIKF3PKKG3MZIUXUFQ)
![version 2](https://raw.githubusercontent.com/Coatll/Chak/main/chak2.png?token=GHSAT0AAAAAACIEASUHC67AVWR3MJHMVJ4YZIUXU3A)

I really hope you'll like it. Can you add it to the game? 
_(I have created a repository for it, I don't know any other way to upload and link images or files here.)_

**Comment by Coatll** [comment]
There is **one irregularity**: I guess, unpromoted pawns of the opponent should be flipped vertically to reflect graphically their rules of movement. So maybe all opponent pieces should be flipped.

If this is a problem for the two-colored armies, I can make both sides of grey color. Just with slightly different shades and with the opposite rotation. Also adding Mayan ornaments at the bottom of the icons to vividly mark their orientation.

**Comment by gbtami** [comment]
This is wonderful! No need to make both sides of grey color at all.
I hope you draw them in .svg (not .pgn), because pixel graphics can't scale well.
PR please! :)

**Comment by CouchTomato87** [comment]
This is great! I actually intended to make something like this, but never got around to it, especially for how much work it is. But you carried on the spirit :)

Regarding promotable pieces, there are two ways to tackle them:
1) Xiangqi-style. The pieces don't actually change. This is how I originally intended it. However, I had to make the pieces slightly different so that we could tell which is which for interface purposes because In Pychess, the promoted and nonpromoted pieces are considered "separate," while in Xiangqi, they are considered the same piece.

2) Flipping pieces. Pieces flip to a promoted side. This is actually also possible given the stone-like pieces. With this, you could be more flexible with how the pieces appear. In this case, you can modify the warrior to properly reflect its movement. For this set, I prefer this option.

**Comment by Coatll** [comment]
Thanks for the positive reaction, guys! 

@gbtami I did a PR, or started it, but there are some strange conflicts with files I never touched. Do I need to resolve them? I almost forgot how to use Git. Had no idea you'd want me to use it when I started to create the pieces.

The images are in svg format. I chose the 1st variation above, since you didn't tell me which one you like better.

@CouchTomato87 OK, I hope you'll make it work. I just wanted to add the images, and not to modify the functionality.

**Comment by gbtami** [comment]
@Coatll btw I like your simple board as well. Can you commit/PR it as well please?
